### DISCUSSION TITLE
Original Gameplay

### DISCUSSION THREAD

**Discussion by RetroGamerFan** [CorsixTH/CorsixTH#2076] [Research Room]
1. The original game allowed setting the heating costs to the lowest setting at the end-of-the-month so the heating effectively is the cheapest it can be without removing radiators.

2. Staff used to stay inside the staff room till both their tiredness and morale levels were replenished. That way you could have trained consultant doctors on a peasant's wage.

3. Radiator mechanics are very different to the original game.

4. Epidemics are much harder to cover-up in the original and you could send patients to a competing AI hospital.

Maybe an "Original Game-Play With Cheese" option is nice for the purists?

**Comment by TheCycoONE** [comment]
I always considered the radiator changes unambiguously better - but I'm open to an original style straight distance version, and naive billing.

The staff room change is interesting and quite possibly unintentional. Are you able to demonstrate this? How does the tiredness policy impact the moral condition?

**Reply by RetroGamerFan** [reply]
So staff get stressed over time. By setting the hospital policy low like 40% tiredness, staff will go to the staff room before they start complaining about a pay-rise. I would have to setup Dosbox or something to replay the original game, but I will do it if you need a save file demonstrating all doctors on the last level working for less than 100 bucks.

Maybe the game was never supposed to be cheesed that way, but it makes me smile thinking doctors (not janitors) end up with the lowest wages.

**Reply by TheCycoONE** [reply]
I'd be looking more for something like a doctor that is no longer tired but staying in the staff room because their moral isn't high enough. A video or a save - and how it behaves at different policy settings.

**Comment by RetroGamerFan** [comment]
Mandela is messing with me, the trainee doctor leaves the staff room before their morale is replenished fully. Still, in the original game you could have the cheapest consultants running the circus without asking for pay rises.

Basically I placed radiators around the doctor in the training room to stress him out quickly.

[HORZA.zip](https://github.com/CorsixTH/CorsixTH/files/7623286/HORZA.zip).

Save file attached - Mandela was right about the cheap heating-bill cheese at least.
